config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  devise_for :users
+  resources :recipe_foods
+  resources :recipes, only: [:index,:new,:create,:show,:destroy]
+  resources :foods 
+  resources :users
```

Comment:
- [ ] It will be great to ensure that the only routes generated, are the ones that are used in the application. You can use either `only` or `expect` to remove the unwanted routes. For example: 
- You have generated all the restful routes for your `recipe_foods` However when i checked the controller, the edit route and the show routes were not used. Kindly fix this too